test/controllers/healthcheck_controller_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'test_helper'
+require 'minitest/mock'
+
+class Api::V0::DiffControllerTest < ActionDispatch::IntegrationTest
+  include Devise::Test::IntegrationHelpers
```

Comment:
Minor note: you shouldn’t need this since you don’t need to do auth for this endpoint (`Devise` is the authentication library we use).